README.md

```diff
@@ -12,13 +12,16 @@ This is the server component of the AeroGear metrics service. It is a RESTful AP
 
 ## Clone and Install Dependencies
 
-1. Clone this repository
+1. Clone this repository to `$GOPATH/src/github.com/aerogear/aerogear-metrics-api`
+```sh
+mkdir -p $GOPATH/src/github.com/aerogear
```

Comment:
+1 That's better if you like to do development and push changes back as it will use ssh remote.